lua/wire/server/wire_map_interface.lua

```diff
@@ -0,0 +1,64 @@
+-- Serverside functionalities of Wire Map Interface
+
+local function validateId(id)
+	id = tonumber(id)
+	if not id then return false end
+	return id == id and id >= -1 and id <= 0xFFFF
```

Comment:
Just realized the first check can be `id ~= nil` and NaN will fail the comparators so no need to check it. 
```suggestion
	return id ~= nil and id >= -1 and id <= 0xFFFF
```